enhancements/sig-policy/98-long-term-compliance-history/README.md

```diff
@@ -81,15 +81,15 @@ string/text column, then those could be used instead of a substring query.
 
 ##### policies Table
 
-| Column           | Description                                                                                                           |
-| ---------------- | --------------------------------------------------------------------------------------------------------------------- |
-| id               | an internal primary key                                                                                               |
-| kind             | a string column of the policy Kind (e.g. ConfigurationPolicy)                                                         |
-| api_group        | a string column of the policy group (e.g. policy.open-cluster-management.io)                                          |
-| name             | a string column for the name of the policy                                                                            |
-| parent_policy_id | an optional foreign key to the parent policy in the managed cluster namespace on the hub in the parent_policies table |
-| spec             | a string column of the spec field of the replicated policy as JSON                                                    |
-| spec_hash        | a string column of a SHA1 hash of the spec field of the replicated policy                                             |
+| Column    | Description                                                                                                                        |
+| --------- | ---------------------------------------------------------------------------------------------------------------------------------- |
+| id        | an internal primary key                                                                                                            |
+| kind      | a string column of the policy Kind (e.g. ConfigurationPolicy)                                                                      |
+| api_group | a string column of the policy group (e.g. policy.open-cluster-management.io)                                                       |
+| name      | a string column for the name of the policy                                                                                         |
+| namespace | an optional string column for the namespace of the policy. This will not be set for native OCM policies to avoid data duplication. |
```

Comment:
Not set on OCM policies because it would just always be the cluster namespace?

Reply:
Exactly.